### ISSUE TITLE
`Twemoji(emoji: "❤️")` yields: `Unable to load asset: packages/twemoji/assets/svg/2764-fe0f.svg`

### ISSUE THREAD

**Issue by asmith26** [bigblackclock/twemoji#16]
Hi there, 

I'm trying to use a [red-heart](https://emojipedia.org/red-heart/) emoji, `Twemoji(emoji: "❤️")`, but I seem to get:

```dart
======== Exception caught by SVG ===================================================================
The following assertion was thrown resolving a single-frame picture stream:
Unable to load asset: packages/twemoji/assets/svg/2764-fe0f.svg

When the exception was thrown, this was the stack: 
#0      PlatformAssetBundle.load (package:flutter/src/services/asset_bundle.dart:237:7)
<asynchronous suspension>
#1      AssetBundle.loadString (package:flutter/src/services/asset_bundle.dart:72:27)
<asynchronous suspension>
#2      AssetBundlePictureProvider._loadAsync (package:flutter_svg/src/picture_provider.dart:546:25)
<asynchronous suspension>
Picture provider: ExactAssetPicture(name: "packages/twemoji/assets/svg/2764-fe0f.svg", bundle: null, colorFilter: null)
Picture key: AssetBundlePictureKey(bundle: PlatformAssetBundle#17d3a(), name: "packages/twemoji/assets/svg/2764-fe0f.svg", colorFilter: null, theme: SvgTheme(currentColor: null, fontSize: 14.0, xHeight: 7.0))
====================================================================================================
```

Many thanks for any help, and for this amazing lib! :)

**Comment by asmith26** [comment]
Interestingly ❤️ does appear to be in assets https://github.com/bigblackclock/twemoji/blob/master/assets/svg/2764.svg

**Comment by vegardan** [comment]
This has already been solved (https://github.com/bigblackclock/twemoji/pull/15), but I am still waiting for approval by @bigblackclock

**Comment by hadi-codes** [comment]
fixed v0.4.3
